polylogue/importers/chatgpt.py

```
@@ -175,32 +177,34 @@ def import_chatgpt_export(
-                    html=html,
-                    html_theme=html_theme,
-                    force=force,
-                    allow_dirty=allow_dirty,
-                    registrar=registrar,
-                )
-            )
+        with convo_path.open("rb") as fh:
+            try:
+                for conv in ijson.items(fh, "item"):
+                    if not isinstance(conv, dict):
+                        raise ValueError
```

Ruff (0.15.9)

[warning] Prefer `TypeError` exception for invalid type

(TRY004)